integration-tests/tests/src/index.ts

```diff
@@ -22,6 +22,8 @@ import { flags } from "realm";
 
 // TODO: Refactor tests to disable this
 flags.ALLOW_VALUES_ARRAYS = true;
+// We need this to call Realm.cleanTestState()
+flags.CLEAN_TEST_STATE = true;
```

Comment:
Typo? `CLEAR_` would align with the method name (rather than `CLEAN_`) 👍

Reply:
Oh my. Thanks.

Reply:
I've updated this and hopefully all other occurrences 🤞